Integration test "test_remove_playlist_bad_id" fails without playlists present

One of the integration tests fail when run against a Sonos S2 speaker, specifically a Sonos 'Move' running software v12.0. Output below.

(The rest of the integration tests pass.)

```
________________________________________________________________________ TestSonosPlaylist.test_remove_playlist_bad_id _________________________________________________________________________

self = <test_integration.TestSonosPlaylist object at 0x107dda490>, soco = SoCo("192.168.0.41")

    def test_remove_playlist_bad_id(self, soco):
        """Test attempting to remove a Sonos playlist using a bad id."""
        # junky bad
        with pytest.raises(SoCoUPnPException):
            soco.remove_sonos_playlist("SQ:-7")
        # realistic non-existing
>       hpl_i = max([int(x.item_id.split(":")[1]) for x in soco.get_sonos_playlists()])
E       ValueError: max() arg is an empty sequence

tests/test_integration.py:487: ValueError
```

--- a/tests/test_integration.py
+++ b/tests/test_integration.py
@@ -484,7 +484,12 @@ class TestSonosPlaylist(object):
         with pytest.raises(SoCoUPnPException):
             soco.remove_sonos_playlist("SQ:-7")
         # realistic non-existing
-        hpl_i = max([int(x.item_id.split(":")[1]) for x in soco.get_sonos_playlists()])
+        playlists = soco.get_sonos_playlists()
+        # Accommodate the case of no existing playlists
+        if len(playlists) == 0:
+            hpl_i = 0
+        else:
+            hpl_i = max([int(x.item_id.split(":")[1]) for x in playlists])
         with pytest.raises(SoCoUPnPException):
             soco.remove_sonos_playlist("SQ:{}".format(hpl_i + 1))
 

